reference/json/BigCommerce_Catalog_API.oas2.json

```diff
@@ -25393,6 +25393,13 @@
             "in": "query",
             "type": "string"
           },
+          {
+            "name": "id",
+            "description":"Get details of multiple specific categories. Include in a comma-seperated list. Use syntax: `?id:in=",
```

Comment:
Unsure about the wording here. 

- "Get details of multiple specific categories." 
Does this imply that you're getting details about the categories (**Suggestion:** "Get details about...) or details within the categories (**Suggestion:** Get details from...)